utils/installer/install.sh

```diff
@@ -262,6 +280,13 @@ EOF
 }
 
 function setup_lvim() {
+  # no need to install in shared_mode
+  if [ -n "$SHARE_NEOVIM_CONFIG" ]; then
+    echo "You can now add 'require \"lvim\"' in your ~/.config/init.lua"
```

Comment:
I think this is incompatible with my changes in 1406, also how would it work if the config requires lvim and lvim loads the config, don't we have an import loop ?
Since we made lvim a "separate instance", I wonder if we want to support this configuration

Reply:
I was afraid they might be incompatible. 

I actually think that this is the only reliable way to use lunarvim on Windows, it's also probably the easiest way to use it with a GUI.

There's no `lvim` binary anymore in this mode, you just start `nvim` like usual and require `lvim` at some point.

Reply:
I'm wondering how cna you require lvim using nvim when it is not in the runtime path, I think my reasons for saying this is incompatible with my changes would be the same for the current implementation. So maybe this works and I'm missing something :)

Note that the path `~/.config/init.lua"` seems incorrect